apps/mobile/src/utils/useSanityMaintenanceCheckMobile.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { AppState, AppStateStatus } from 'react-native';
+
+import { useSanityMaintenanceCheck } from './sanity';
+
+export function useSanityMaintenanceCheckMobile() {
+  const {
+    fetchMaintenanceModeStatus,
+    maintenanceCheckLoadedOrError,
+    currentlyActiveMaintenanceNoticeContent,
+  } = useSanityMaintenanceCheck();
```

Comment:
this file was renamed, and maintenance mode fetching business logic was moved to shared util